scripts/table_diff/changes/changes_test.go

```diff
@@ -67,52 +87,40 @@ func Test_getChanges(t *testing.T) {
 					Breaking: true,
 				},
 				{
-					Text:     "Table `azure_appservice_functions`: column added with name `site_id` and type `String`",
-					Breaking: false,
+					Text: "Table `azure_appservice_functions`: column added with name `site_id` and type `String`",
```

Comment:
Let's not remove the setting of `Breaking` to `false`, I think it's better to be explicit with the test data, otherwise someone reading the test will have to go to the struct definition to understand what is being tested.
Also it's unrelated to the change in this PR and it makes it hard to review the new test